physics/sfc_drv_ruc.F90

```diff
@@ -419,7 +419,7 @@ subroutine lsm_ruc_run                                            & ! inputs
       if(flag_init .and. iter==1) then
       ! Initialize the RUC soil levels, needed for cold starts and warm starts
         CALL init_soil_depth_3 ( zs , dzs , lsoil_ruc )
-        xlai = 0.
+        if (.not. restart) !xlai = 0.
```

Comment:
Please don't make any changes while I am trying to fix the issue, if possible. Also, this change means that `xlai` is never set to zero (beecause of the `!`), correct?

Reply:
This code change doesn't compile, because restart is not known here. I'll fix it.

Reply:
Ok, I will not do anything. I removed the '!" in the next commit.

Reply:
Thanks, I pulled this change as well. If you want me to try something, please just let me know. Pulling in new changes from the PR means I need to copy my version of the file (with my changes) to a temporary file, then checkout the original file, then pull in your updates, and then apply my uncommitted changes manually again.